recipe/meta.yaml

```diff
@@ -21,14 +21,14 @@ build:
 
 requirements:
   build:
-    - python
+    - python ==python2.7
```

Comment:
This should not be needed since you have a skip statement above to explicitly avoid Python 3.